docs/reference/objects/product/extra.md

```diff
@@ -50,6 +58,8 @@ boolean
 {: .label .fs-1 }
 
 Returns `true` if the extra has unlimited stock.
+- When accessed through an [experience date]({% link docs/reference/objects/product/experience_date.md %}), it will return `true` if the extra has unlimited stock on the given date.
+- When accessed independently or through a product, it will return `true` if the extra has unlimited stock on at least one of the product dates.
```

Comment:
Following on from the slack convo just want to clarify, would this also be for just upcoming dates?

```suggestion
- When accessed independently or through a product, it will return `true` if the extra has unlimited stock on at least one of the upcoming dates.
```

Reply:
Yes - good spot!